docs/source/reference/integration.rst

```diff
@@ -144,8 +135,6 @@ We summarize the necessary dependencies for each integration.
 +-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------+
 | `Catalyst <https://github.com/optuna/optuna/blob/master/optuna/integration/catalyst.py>`_                                                                                         | catalyst                           |
 +-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------+
-| `CatBoost <https://github.com/optuna/optuna/blob/master/optuna/integration/catboost.py>`_                                                                                         | catboost                           |
-+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------+
```

Comment:
Everything moved to optuna-integration is still here, e.g., shap, AllenNLP, so I think we can keep it.

Reply:
Fixed.